service/controller/resource/app/desired.go

```diff
@@ -70,7 +70,12 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) ([]*g8s
 
 	// Define app CR for app-operator in the management cluster namespace.
 	appOperatorAppSpec := newAppOperatorAppSpec(cr, appOperatorComponent)
-	apps = append(apps, r.newApp(uniqueOperatorVersion, cr, appOperatorAppSpec, g8sv1alpha1.AppSpecUserConfig{}))
+	apps = append(apps, r.newApp(uniqueOperatorVersion, cr, appOperatorAppSpec, g8sv1alpha1.AppSpecUserConfig{
+		ConfigMap: g8sv1alpha1.AppSpecUserConfigConfigMap{
+			Name:      "app-operator-konfigure",
```

Comment:
I checked all installations and we basically have this configmap all over the place.

Reply:
Yes, it needs to be there for the MC app-operator so I think its safe to rely on it being there.